.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": ["airbnb", "bliss", "prettier"],
+  "rules": {
+    "class-methods-use-this": "warn",
+
+    "import/no-dynamic-require": 0,
```

Comment:
Using the error/warn/info enums is better for eslint configs

Reply:
Do you mean error/warn/**off**?

Reply:
oh yea that